js/build.js

```diff
@@ -2,6 +2,8 @@ window.ui = window.ui || {};
 Fliplet.Widget.instance('list-thumb-l', function(data) {
   var $container = $(this);
 
+  $container.translate({ swipeToSaveLabel: data.swipeToSaveLabel || T('widgets.list.largeThumbs.defaultListName') });
```

Comment:
I suggest declaring a `swipeToSaveLabel` variable so that it can be used in other places and avoid repetition.

```suggestion
  var swipeToSaveLabel = data.swipeToSaveLabel || T('widgets.list.largeThumbs.defaultListName');
  
  $container.translate({ swipeToSaveLabel: swipeToSaveLabel });
```